lib/msal-common/src/crypto/PopTokenGenerator.ts

```diff
@@ -35,20 +18,6 @@ export class PopTokenGenerator {
         this.cryptoUtils = cryptoUtils;
     }
 
-    async generateCnf(request: SignedHttpRequestParameters): Promise<string> {
```

Comment:
Arguably a breaking change, given you are removing a public api from an exported class.

Reply:
I hadn't entirely considered this. Should we remove the `KeyManager` class entirely and keep using `PoPTokenGenerator` even if it becomes a misnomer? Truth is `KeyManager` only sort of wraps calls to `CryptoOps` anyway.

Reply:
Kept generateCnf in PopTokenGenerator and also moved CryptoKeyManager APIs into PopTokenGenerator.